src/marker.js

```diff
@@ -31,7 +31,7 @@ function markerArrow(color, context) {
     .attr("viewBox", "-5 -5 10 10")
     .attr("markerWidth", 6.67)
     .attr("markerHeight", 6.67)
-    .attr("orient", "auto")
+    .attr("orient", "auto-start-reverse")
```

Comment:
This is especially noticeable with tick and rule (as shown in the tests); I think this is a better default for the *arrow* marker.

Reply:
Agree it's a better default for rules and ticks, that indicate an extent; but for a line that indicates a sequence, it feels a bit wrong if the first arrow is pointing outwards.

Reply:
It feels less wrong than having the arrows point the same way for a rule and a line. I don’t see any practical demand for having an arrow at the start of the line; you’d typically use markerMid instead, and possible markerEnd.

The alternative is we have to introduce two types of marker, one for line (*arrow* if we retain backwards compatibility) which will almost never be used for that purpose, and another for rule and tick (*arrow-reverse*, say) which won’t be as discoverable.

Reply:
I’ve reverted the changes to the *arrow* marker and added an *arrow-reverse* marker in the latest commit.